Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+    name: "LaunchAtLogin",
+    platforms: [
+        .macOS(.v10_12)
+    ],
+    products: [
+        .library(
+            name: "LaunchAtLogin",
+            targets: ["LaunchAtLogin"]
+        )
+    ],
+    targets: [
+        .target(
+            name: "LaunchAtLogin",
+            path: "LaunchAtLogin",
```

Comment:
Can you just use the `Sources` directory structure here (also update in Xcode) so we are more prepared for only-SPM support and then we don't need to specify this property.